<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenDialog</class>
 <widget class="QDialog" name="OpenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打开文件对话框 / ImageZhuo</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="edt_width"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>读取器</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="combo_reader"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>高</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="combo_pixelFormat"/>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="lbl_hourglass">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="OpenDialog.qrc">:/icon/asset/hourglass.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="btn_done">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>宽</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>像素格式</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edt_height"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>文件路径</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edt_filePath"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn_browse">
      <property name="text">
       <string>浏览..</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>combo_reader</tabstop>
  <tabstop>edt_width</tabstop>
  <tabstop>edt_height</tabstop>
  <tabstop>combo_pixelFormat</tabstop>
  <tabstop>btn_done</tabstop>
 </tabstops>
 <resources>
  <include location="OpenDialog.qrc"/>
 </resources>
 <connections/>
</ui>
